### PR TITLE
Add units tests

### PR DESCRIPTION
**Pull request by haseo29**
### Added
- Add units tests for SonataGroup and CreateUserCommand .
-Update Composer.json.
### Changed
-Composer.json : Add dependency required

### CODE REVIEW

File: tests/Functional/SonataGroupTest.php [hunk 1]
```diff
@@ -4,256 +4,242 @@
...
-
 /**
  * Generated by PHPUnit_SkeletonGenerator on 2017-05-10 at 17:49:31.
  */
-
 use PHPUnit\Framework\TestCase;
 
-class SonataGroupTest extends TestCase
-{
+class SonataGroupTest extends TestCase {
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Wrong codestyle.

You must have a line feed before opening brackets

File: tests/Functional/SonataGroupTest.php [hunk 2]
```diff
@@ -4,256 +4,242 @@
...
-    
+
     /**
      * Sets up the fixture, for example, opens a network connection.
      * This method is called before a test is executed.
      */
-    protected function setUp()
-    {    
-        $this->object = new SonataGroup;   
+    protected function setUp() {
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: For methods it's ok

File: tests/Functional/SonataGroupTest.php [hunk 3]
```diff
@@ -4,256 +4,242 @@
...
      * @covers Librinfo\SonataSyliusUserBundle\Entity\SonataGroup::__toString
      * @todo   Implement test__toString().
      */
-    public function test__toString()
-    {
+    public function test__toString() {
         // Remove the following lines when you implement this test.
         $this->markTestIncomplete(
-            'This test has not been implemented yet.'
+                'This test has not been implemented yet.'
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Wrong number of spaces for indentation (must be 4)

File: tests/Functional/SonataGroupTest.php [hunk 4]
```diff
@@ -4,256 +4,242 @@
...
      * @covers Librinfo\SonataSyliusUserBundle\Entity\SonataGroup::getId
      * @todo   Implement testGetId().
      */
-    public function testGetId()
-    {
+    public function testGetId() {
         // Remove the following lines when you implement this test.
         $this->markTestIncomplete(
-            'This test has not been implemented yet.'
+                'This test has not been implemented yet.'
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Wrong number of spaces for indentation (must be 4)

File: tests/Functional/SonataGroupTest.php [hunk 5]
```diff
@@ -4,256 +4,242 @@
...
      * @covers Librinfo\SonataSyliusUserBundle\Entity\SonataGroup::getName
      * @todo   Implement testGetName().
      */
-    public function testGetName()
-    {
+    public function testGetName() {
         // Remove the following lines when you implement this test.
         $this->markTestIncomplete(
-            'This test has not been implemented yet.'
+                'This test has not been implemented yet.'
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Wrong number of spaces for indentation (must be 4)

File: tests/Functional/SonataGroupTest.php [hunk 6]
```diff
@@ -4,256 +4,242 @@
...
      * @covers Librinfo\SonataSyliusUserBundle\Entity\SonataGroup::hasRole
      * @todo   Implement testHasRole().
      */
-    public function testHasRole()
-    {
+    public function testHasRole() {
         // Remove the following lines when you implement this test.
         $this->markTestIncomplete(
-            'This test has not been implemented yet.'
+                'This test has not been implemented yet.'
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Wrong number of spaces for indentation (must be 4)

File: tests/Functional/SonataGroupTest.php [hunk 7]
```diff
@@ -4,256 +4,242 @@
...
      * @covers Librinfo\SonataSyliusUserBundle\Entity\SonataGroup::getRoles
      * @todo   Implement testGetRoles().
      */
-    public function testGetRoles()
-    {
+    public function testGetRoles() {
         // Remove the following lines when you implement this test.
         $this->markTestIncomplete(
-            'This test has not been implemented yet.'
+                'This test has not been implemented yet.'
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Wrong number of spaces for indentation (must be 4)

File: tests/Functional/SonataGroupTest.php [hunk 8]
```diff
@@ -4,256 +4,242 @@
...
      * @covers Librinfo\SonataSyliusUserBundle\Entity\SonataGroup::setName
      * @todo   Implement testSetName().
      */
-    public function testSetName()
-    {
+    public function testSetName() {
         // Remove the following lines when you implement this test.
         $this->markTestIncomplete(
-            'This test has not been implemented yet.'
+                'This test has not been implemented yet.'
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Wrong number of spaces for indentation (must be 4)

File: tests/Functional/SonataGroupTest.php [hunk 9]
```diff
@@ -4,256 +4,242 @@
...
      * @todo   Implement testSetRoles().
      */
-    public function testSetRoles()
-    {          
+    public function testSetRoles() {
         // Remove the following lines when you implement this test.
         $this->markTestIncomplete(
-            'This test has not been implemented yet.'
-        ); 
+                'This test has not been implemented yet.'
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Wrong number of spaces for indentation (must be 4)

File: tests/Functional/SonataGroupTest.php [hunk 10]
```diff
@@ -4,256 +4,242 @@
...
 
     /**
      * @covers Librinfo\SonataSyliusUserBundle\Entity\SonataGroup::addUser
      * @todo   Implement testAddUser().
      */
-    public function testAddUser()
-    {               
+    public function testAddUser() {
+        $this->markTestSkipped(
+                'need call to addGroup'
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Wrong number of spaces for indentation (must be 4)

File: tests/Functional/SonataGroupTest.php [hunk 11]
```diff
@@ -4,256 +4,242 @@
...
 
     /**
      * @covers Librinfo\SonataSyliusUserBundle\Entity\SonataGroup::removeUser
      * @todo   Implement testRemoveUser().
      */
-    public function testRemoveUser()
-    {
+    public function testRemoveUser() {
+        $this->markTestSkipped(
+                'need call to removeGroup'
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Wrong number of spaces for indentation (must be 4)

File: tests/Functional/SonataGroupTest.php [hunk 12]
```diff
@@ -4,256 +4,242 @@
...
      * @covers Librinfo\SonataSyliusUserBundle\Entity\SonataGroup::getUsers
      * @todo   Implement testGetUsers().
      */
-    public function testGetUsers()
-    {
+    public function testGetUsers() {
         // Remove the following lines when you implement this test.
         $this->markTestIncomplete(
-            'This test has not been implemented yet.'
+                'This test has not been implemented yet.'
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Wrong number of spaces for indentation (must be 4)

File: tests/Functional/SonataGroupTest.php [hunk 13]
```diff
@@ -4,256 +4,242 @@
...
      * @covers Librinfo\SonataSyliusUserBundle\Entity\SonataGroup::setUsers
      * @todo   Implement testSetUsers().
      */
-    public function testSetUsers()
-    {
+    public function testSetUsers() {
         // Remove the following lines when you implement this test.
         $this->markTestIncomplete(
-            'This test has not been implemented yet.'
+                'This test has not been implemented yet.'
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Wrong number of spaces for indentation (must be 4)

File: tests/Functional/SonataGroupTest.php [hunk 14]
```diff
@@ -4,256 +4,242 @@
...
-    {
+    public function testRemoveUser() {
+        $this->markTestSkipped(
+                'need call to removeGroup'
+        );
+
         $toto = 'lundi matin';
-        $user = $this->getMockForAbstractClass(AdvancedUserInterface::class);
+        $user = $this
+                ->getMockForAbstractClass(AdvancedUserInterface::class);
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Wrong number of spaces for indentation (must be 4)

File: tests/Functional/SonataGroupTest.php [hunk 15]
```diff
@@ -4,256 +4,242 @@
...
      * @covers Librinfo\SonataSyliusUserBundle\Entity\SonataGroup::setId
      * @todo   Implement testSetId().
      */
-    public function testSetId()
-    {
+    public function testSetId() {
         // Remove the following lines when you implement this test.
         $this->markTestIncomplete(
-            'This test has not been implemented yet.'
+                'This test has not been implemented yet.'
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Wrong number of spaces for indentation (must be 4)

File: tests/Functional/SonataGroupTest.php [hunk 16]
```diff
@@ -4,256 +4,242 @@
...
      * @covers Librinfo\SonataSyliusUserBundle\Entity\SonataGroup::isNew
      * @todo   Implement testIsNew().
      */
-    public function testIsNew()
-    {
+    public function testIsNew() {
         // Remove the following lines when you implement this test.
         $this->markTestIncomplete(
-            'This test has not been implemented yet.'
+                'This test has not been implemented yet.'
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Wrong number of spaces for indentation (must be 4)

File: tests/Functional/BundleTest.php [hunk 17]
```diff
@@ -17,9 +17,9 @@ protected function setUp() {
     }
 
     public function testServicesAreInitializable() {
-        
+
         $this->markTestSkipped(
-        'test class'
+                'test class'
```
Comment (PapsOu) [inline tests/Functional/BundleTest.php]: Not good again, there is 8 spaces, only 4 spaces expected

File: tests/Functional/CreateUserCommandTest.php [hunk 18]
```diff
@@ -14,96 +14,45 @@ class CreateUserCommandTest extends TestCase {
...
 
     /**
      * @covers Librinfo\SonataSyliusUserBundle\Command\CreateUserCommand::configure
      * @todo   Implement testConfigure().
      */
     public function testConfigure() {
-        
         $this->markTestSkipped(
-        'need call to configure'
+                'need call to configure'
```
Comment (PapsOu) [inline tests/Functional/CreateUserCommandTest.php]: Not good again, there is 8 spaces, only 4 spaces expected

File: tests/Functional/SonataGroupTest.php [hunk 19]
```diff
@@ -186,14 +180,11 @@ public function testRemoveUser() {
         $toto = 'lundi matin';
         $user = $this
                 ->getMockForAbstractClass(AdvancedUserInterface::class);
```
Comment (PapsOu) [inline tests/Functional/SonataGroupTest.php]: Not good again, there is 8 spaces, only 4 spaces expected

**Review by PapsOu — COMMENTED** [review]
Please check you IDE code style

**Comment by FanchTheSystem** [comment]
please remove *~ file from this pull request

**Review by FanchTheSystem — APPROVED** [review]

**Review by PapsOu — APPROVED** [review]

**Pull request merged**